### PR TITLE
Fix issues with radius

### PR DESCRIPTION
**Pull request by Firstyear**
Also cleans some noise with logs

### CODE REVIEW

File: pykanidm/kanidm/__init__.py [hunk 1]
```diff
@@ -165,6 +165,7 @@ async def _call(
                 elif "authorization" not in headers:
                     logging.debug("Setting auth headers as Authorization not in keys")
                     headers.update(self._token_headers)
+            logging.debug("headers --> %s" % headers)
```
Comment (yaleman) [inline pykanidm/kanidm/__init__.py:168]: I specifically wasn't logging this because it leaks auth tokens, and that's bad...

**Review by yaleman — CHANGES_REQUESTED** [review]
Security note, but we also need to update the docs to reflect the change.

File: pykanidm/kanidm/__init__.py (hunk 1 above)
Comment (Firstyear) [reply pykanidm/kanidm/__init__.py]: Fixed!

**Comment by Firstyear** [comment]
Doc fixes :)

**Review by yaleman — APPROVED** [review]
I touched it lasts, so I might as well approve it! 🦈

**Pull request merged**